src/test-utils/dom/button-dropdown/index.ts

```diff
@@ -40,20 +40,30 @@ export default class ButtonDropdownWrapper extends ComponentWrapper {
    * Finds an item in the open dropdown by item id. Returns null if there is no open dropdown.
    *
    * This utility does not open the dropdown. To find dropdown items, call `openDropdown()` first.
+   *
```

Comment:
There are many button dropdown tests that use expandToViewport (see example: https://github.com/cloudscape-design/components/blob/main/src/button-dropdown/__tests__/button-dropdown-highlighted.test.tsx#L24), but the expandToViewport option wasn't needed. I think that is because the `findOpenDropdown()` method searches from the root, not component root - assuming there is at most one open dropdown. Why do we need the expandToViewport option now?

Reply:
Good call-out. This change wasn't needed.